src/styles/images/index.md.njk

```diff
@@ -28,7 +28,7 @@ If your image represents something physical, such as a letter, document or credi
 
 ## Alternative text
 
-This guidance is for government teams that build online services. [To learn about publishing alternative text in GOV.UK mainstream content, go to GOV.UK's 'Images' guidance.](https://www.gov.uk/guidance/content-design/images)
+This guidance is for government teams that build online services. [To learn about publishing alternative text as part of information and guidance on the GOV.UK website, go to GOV.UK's 'Images' guidance.](https://www.gov.uk/guidance/content-design/images)
```

Comment:
Convos in progress still (hopefully nearly there). We're considering that for this type of user (content people and service teams), the way this content published is also important distinction, as well as the type of content.

```suggestion
This guidance is for government teams that build online services. [To learn about writing alternative text for website pages on www.gov.uk, using the main GOV.UK publishing platform, go to GOV.UK’s ‘Images’ guidance.](https://www.gov.uk/guidance/content-design/images)
```